test/driver-test/driver-test.cpp

```diff
@@ -0,0 +1,306 @@
+/*
+* driver-test.cpp
+* Nick DeMarco
+* (c) 2020 sigcpp https://sigcpp.github.io. See LICENSE.MD
+*
+* Attribution and copyright notice must be retained.
+* - Attribution may be augmented to include additional authors
+* - Copyright notice cannot be altered
+* Attribution and copyright info may be relocated but they must be conspicuous.
+*
+* Tester for test-stl-lite driver
+*/
+
+#include <iostream>
+#include <vector>
+#include <string>
+
+#include "../verifiers.h"
+#include "../options.h"
+#include "../tester.h"
+#include "../utils.h"
+
+void driver_test();
+void test_get_options();
+void compare(const Options& received, const Options& expected, const std::string& test_case);
+void parse_cmd_line_and_compare(char* args[], const std::string& cmd_line, const Options& expected, const std::string& test_case);
+
+//unit test for the test driver
+void driver_test()
+{
+	test_get_options();
+}
+
+void parse_cmd_line_and_compare(const std::string& cmd_line, const Options& expected,
+	const std::string& test_case)
+{
+	//this is meant to replicate the main() parameter char* argv[]
```

Comment:
The use of "this" needs a referent, without which we wouldn't what the comment is about

```suggestion
	//args mimics parameter the argv parameter in main
```

Reply:
I think there is an extra "parameter" in your suggestion.

Did you mean "args mimics the argv parameter in main"?

I will push that change but feel free to add another suggestion if you meant something else.